inyoka_theme_default/templates/portal/usercp/subscriptions.html

```diff
@@ -0,0 +1,108 @@
+{#
+    portal/usercp/subscriptions.html
+    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+    This page shows all subscriptions of the user and let him
+    manage them.
+
+    :copyright: (c) 2013-2015 by the Inyoka Team, see AUTHORS for more details.
+    :license: BSD, see LICENSE for more details.
+#}
+
+{% extends 'portal/usercp/overall.html' %}
+
+{% block breadcrumb %}
+  {{ super() }}
+
+  {{ macros.breadcrumb_item(_('Subscriptions'), href('portal', 'usercp', 'subscriptions'))}}
+{% endblock %}
+
+{% block sidebar %}
+  {{ super() }}
+
+  {# FIXME Waiting for changes in backend #}
+  <h3>{% trans %}Filter{% endtrans %}</h3>
+  <div class="container noh3">
+    {{ filtersets[0].form.content_type }}
+  </div>
+{% endblock %}
+
+
+{% block content %}
+  {% set rendered_pagination = macros.render_pagination(pagination) %}
+
+  <h3>{% trans %}Subscriptions{% endtrans %}</h3>
+  {% call macros.outer_form(csrf_token(), button_visible=False, manually_rendered=True) %}
+    <div class="pagination right">{{ rendered_pagination }}</div>
+    <table class="usercp_subscriptions">
+      <thead>
+        <tr>
+          <th class="checkboxes"><input type="checkbox" id="check_all" title="{% trans %}(De)select all{% endtrans %}" /></th>
+          <th class="name">{% trans %}Name{% endtrans %}</th>
+          <th class="read">{{ table.get_html('notified', _('Read')) }}</th>
+          <th class="status">{% trans %}State{% endtrans %}</th>
+        </tr>
+      </thead>
+      <tbody>
+        {% for sub in subscriptions %}
+          {% set type = sub.content_type.model %}
+          {% set object = sub.content_object %}
+          {% if type != 'suggestion' and object %}
+            <tr>
+              <td class="checkboxes"><input type="checkbox" name="select" value="{{ sub.id }}" /></td>
+              <td class="items{% if not sub.can_read %} unreadable{% endif %}">
+                {{ object._meta.verbose_name }}: <a href="{{ object|url }}">{{ object|e }}</a>
+                {% if type == 'topic' %}
+                    <a class="fa_icon-share" href="{{ object|url('first_unread') }}" title="{% trans %}Show unread posts{% endtrans %}"></a>
```

Comment:
it would be great to render the title-tags as a tooltip (if possible). Mobile would be another problem, as there is no real hover…